worker/jobTypes/productionDeployJob.js

```diff
@@ -8,10 +8,10 @@ const buildTimeout = 60 * 450;
 const invalidJobDef = new Error('job not valid');
 
 async function verifyUserEntitlements(currentJob) {
-  const user = currentJob.user;
+  const [user] = currentJob.user;
```

Comment:
I'm confused about the use of destructuring here -- have we changed the typing around these objects?